fala/templates/generic_base.html

```diff
@@ -0,0 +1,68 @@
+{%- from "govuk_frontend_jinja/components/skip-link/macro.html" import govukSkipLink -%}
```

Comment:
This file is not using GOVUK templates. You wouldn't have to make your own when you can just use one straight from GOVUK itself. 

I'd also rename the file to just `base.html`